cvat-canvas3d/src/typescript/canvas3dView.ts

```diff
@@ -750,12 +754,27 @@ export class Canvas3dViewImpl implements Canvas3dView, Listener {
         return cuboid;
     }
 
-    private setupObjects(): void {
+    private setupObjects(selected?: boolean): void {
+        const { clientID } = this.model.data.activeElement;
         if (this.views.perspective.scene.children[0]) {
-            this.clearSceneObjects();
+            if (!selected) {
+                this.clearSceneObjects();
+            }
+            this.resetColor();
             this.setHelperVisibility(false);
             for (let i = 0; i < this.model.data.objects.length; i++) {
                 const object = this.model.data.objects[i];
+                if (selected) {
+                    const target = this.views.perspective.scene.getObjectByName(clientID);
+                    if (`${object.clientID}` !== clientID) {
+                        continue;
+                    } else {
+                        this.views.perspective.scene.children[0].remove(target);
+                        this.views.side.scene.children[0].children = [];
+                        this.views.front.scene.children[0].children = [];
+                        this.views.top.scene.children[0].children = [];
```

Comment:
Why do we need to clear side/front/top scenes a lot of times if only one object can be there at the same time?